backend/src/main/java/gov/cdc/usds/simplereport/api/accountrequest/AccountRequestController.java

```diff
@@ -56,11 +88,90 @@ public void submitAccountRequest(@Valid @RequestBody AccountRequest body) throws
     if (LOG.isInfoEnabled()) {
       LOG.info("Account request submitted: {}", objectMapper.writeValueAsString(body));
     }
-    emailService.send(sendGridProperties.getAccountRequestRecipient(), subject, body);
-    emailService.send(
+    _es.send(sendGridProperties.getAccountRequestRecipient(), subject, body);
+    _es.send(
         body.getEmail(),
         "Next Steps for SimpleReport",
         "account-next-steps",
         "simplereport-site-onboarding-guide.pdf");
+
+    Map<String, String> reqVars =
+        body.toTemplateVariables().entrySet().stream()
+            .collect(
+                HashMap::new, 
+                (m,e)->m.put(
+                    e.getKey(), 
+                    e.getValue() == null 
+                    ? null 
+                    : e.getValue().toString()), 
+                HashMap::putAll);
+
+    List<DeviceType> devices = _dts.fetchDeviceTypes();
+    Map<String, String> deviceNamesToIds = 
+        devices.stream().collect(Collectors.toMap(d -> d.getName(), d -> d.getInternalId().toString()));
+    Map<String, String> deviceModelsToIds = 
+        devices.stream().collect(Collectors.toMap(d -> d.getModel(), d -> d.getInternalId().toString()));
+
+    List<String> testingDevicesSubmitted = 
+        new ArrayList<>(Arrays.asList(reqVars.get("testingDevices").split(", ")));
+    testingDevicesSubmitted.removeIf(d -> d.toLowerCase().startsWith("other"));
```

Comment:
Is it possible to store this "other" field somewhere? Right now it seems like we're dropping the value, so that when the user goes in to see their org the device won't appear.
Also, if the user has only selected "other" will the graphQL throw an error? I assume organizations need at least one device.

Reply:
a) actually by design, yes, we're not storing the "other" device. it gets passed onto the account request email (in account-request.html) but from a SimpleReport app perspective, there's not much we can do to accommodate those "other" devices anyway at the moment. only the "pre-approved" devices get stored.
b) the sign-up form forces the user to choose a default device from the list of pre-approved devices, and the app enforces that the default device has to be in the user's list of devices, so effectively the answer is yes, the app would throw an error. but it'll come back to the user as a non-descript 5xx error since it's not wired through graphQL like our core app APIs :-\

Reply:
Ahhh ok, I didn't realize that the signup form forces users to choose a default device from the list of acceptable devices, that makes much more sense. Thanks for the detailed explanation!